core/chain-configs/src/client_config.rs

```diff
@@ -493,9 +493,9 @@ impl ClientConfig {
             state_sync: StateSyncConfig::default(),
             transaction_pool_size_limit: None,
             enable_multiline_logging: false,
-            state_split_config: MutableConfigValue::new(
-                StateSplitConfig::default(),
-                "state_split_config",
+            resharding_config: MutableConfigValue::new(
+                ReshardingConfig::default(),
+                "resharding_config",
```

Comment:
Is it safe to change this? Wouldn't we have to go back and rewrite all configs renaming `state_split_config` to `resharding_config`?

Reply:
What configs exactly do you have in mind by "all configs"? It's true that it's best to get this included in the same release as resharding, see the description on this PR, I hope we can still make it.

Reply:
discussed offline
it's fine to do this here because it's a new config that shouldn't be present in any existing config files